app/js/views/reminders.jsx

```diff
@@ -104,6 +107,10 @@ export default class Reminders extends React.Component {
       });
   }
 
+  onParsingFailure() {
+    this.speechController.speak(`I didn't understand that. Can you repeat?`);
```

Comment:
I'm not convinced we need an issue for that. That'll be one item in the task list of the lists feature implementation. Let's wait until the scope of this new feature is more defined.